x-pack/plugins/security_solution/common/api/search_strategy/model/request_paginated_options.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { sort } from '../hosts/model/sort';
+import { pagination } from './pagination';
+import { requestBasicOptionsSchema } from './request_basic_options';
+
+export const requestOptionsPaginatedSchema = requestBasicOptionsSchema
+  .extend({
+    pagination,
+    sort,
+  })
+  .passthrough();
```

Comment:
Why the passthrough?

Reply:
removed! just a temp soution to a failing test